tasks/main.yml

```diff
@@ -155,7 +155,7 @@
 
 - include: "devbox.yml"
   when:
-    - "atom_devbox|bool"
+    - "(atom_devbox is defined and atom_devbox|bool) or atom_environment_type == 'development'"
```

Comment:
I'm not sure how to proceed here. This would deprecate `atom_devbox` until we want to remove it but, since we don't have this documented anywhere and we won't show any warning, I think it's better to just remove it directly. Although, I'm not completely sure where this may be used (AFAIK, [am-packbuild](https://github.com/artefactual-labs/am-packbuild/blob/stable/1.12.x/packer/templates/vagrant-box-atom/provisioning/host_vars/all#L11) and [deploy-pub](https://github.com/artefactual/deploy-pub/blob/master/playbooks/atom-focal/Vagrantfile#L42)).

@mamedin, @sbreker ... thoughts?